server/routes/api.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express')
+const router = express.Router()
+const axios = require('axios')
+const moment = require ("moment")
+const City = require("../model/City")
+
+const apiKey = "f2d2c49051fb06b13f554f1bea9d785e"
+const getUrl = function (cityName) {
+    return `http://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=metric&appid=${apiKey}`
+}
+const cityObj = function (cityData) {
+    let mapedData = {
+        name: cityData.data.name.toLowerCase(),
+        temperature: cityData.data.main.temp,
+        condition: cityData.data.weather[0].description,
+        icon: cityData.data.weather[0].icon,
+        updatedAt:  moment(new Date()).format('LLLL')
+    }
+    return mapedData
+}
+    router.get('/city/:cityName', async function (req, res) {
+        const cityName = req.params.cityName
+        const lon = req.query.lon
```

Comment:
Use object destructuring here:
`const { lat, lon } = req.query`

Same outcome, one line shorter (in this case, sometime it can save you many lines).